### PR TITLE
Expanded model zoo + fixed links in TF-QAT toolkit (w/ sign-off)

### PR DESCRIPTION
**Pull request by gcunhase**
- Added Inception-v3 support: code and results
- Fixed broken links
- Fixed MobileNet test case

Signed-off-by: Gwena Cunha <gwena.cs@gmail.com>

### CODE REVIEW

File: tools/tensorflow-quantization/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # <span style="color:green"> **NVIDIA TensorFlow 2.x Quantization** </span>
 
 This TensorFlow 2.x Quantization toolkit quantizes (inserts Q/DQ nodes) TensorFlow 2.x Keras models for Quantization-Aware Training (QAT).
-We follow NVIDIA's QAT recipe, which leads to optimal model acceleration with [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html) on NVIDIA GPUs and hardware accelerators.
+We follow NVIDIA's QAT recipe, which leads to optimal model acceleration with [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html) on NVIDIA GPUs and hardware accelerators.
```
Comment (kevinch-nv) [inline tools/tensorflow-quantization/README.md:4]: There are a few extra whitespaces added in this commit. Can you remove them?

File: tools/tensorflow-quantization/README.md [hunk 2]
```diff
@@ -60,17 +60,17 @@ If all tests pass, installation is successful.
...
 3. Saving the quantized version of a few layers may not be supported in `TensorFlow < 2.8`:
    - `DepthwiseConv2D` support was added in TF 2.8.
-   - `Conv2DTranspose` is not yet supported by TF (see the open bug [here](https://github.com/tensorflow/model-optimization/issues/964)).
+   - `Conv2DTranspose` is not yet supported by TF (see the open bug [here](https://github.com/tensorflow/model-optimization/issues/964)). 
        However, there's a workaround if you do not need the TF2 SavedModel file and just the ONNX file:
-       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [User Guide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
+       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [userguide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
```
Comment (kevinch-nv) [inline tools/tensorflow-quantization/README.md:73]:
```suggestion
       1. Implement `Conv2DTransposeQuantizeWrapper`. See our [user guide](https://docs.nvidia.com/deeplearning/tensorrt/tensorflow-quantization-toolkit/docs/docs/add_new_layer_support.html#example) for more information on how to do that.
```

Comment (gcunhase) [reply tools/tensorflow-quantization/README.md:73]: I'm okay with that, we went with `userguide` over `User Guide` (or a variation of that) to match the [pytorch-quantization README](https://github.com/NVIDIA/TensorRT/tree/main/tools/pytorch-quantization#resources). Please let us know which one is preferred.

File: tools/tensorflow-quantization/README.md (hunk 1 above)
Comment (gcunhase) [reply tools/tensorflow-quantization/README.md]: Done

File: tools/tensorflow-quantization/README.md (hunk 2 above)
Comment (kevinch-nv) [reply tools/tensorflow-quantization/README.md:73]: Prefer `user guide`

Comment (gcunhase) [reply tools/tensorflow-quantization/README.md]: Noted and updated.

**Review by kevinch-nv — APPROVED** [review]

**Pull request merged**